store/mongo/migration_0_2_0_test.go

```diff
@@ -37,86 +42,161 @@ func TestMigration_0_2_0(t *testing.T) {
 	}{
 		"single dev": {
 			inDevs: []interface{}{
-				model.Device{
+				legacyDevice{
```

Comment:
I am having seconds thoughts about that. technically we are using a different structure in tests than in actual production code, right?

Reply:
This is solely to be able to check that the previous definition of a device - one where the attributes does not have scopes - is correctly migrated to the updated definition. The updated Device can no longer marshal attributes without already putting the scope-name as part of the attribute field, hence the `legacyDevice` definition. (Note that this definition is defined locally and only used for this particular migration test).

Reply:
ok. I trust you, you see?